pkg/machine/qemu/machine.go

```diff
@@ -921,7 +961,11 @@ func (v *MachineVM) stopLocked() error {
 	}
 
 	fmt.Println("Waiting for VM to exit...")
-	for isProcessAlive(vmPid) {
+	for {
```

Comment:
is it possible that we never exit this loop

Reply:
Added retry limit